### PR TITLE
Added parent interaction Id -changes for Mat 58

### PR DESCRIPTION
**Pull request by lbhssandilya**
Added parent Interaction Id In GetTenancyManagementInteraction , To get the issues of a meeting Mat-58

### CODE REVIEW

File: ManageATenancyAPI/Controllers/v2/TraController.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public async Task<IActionResult> Post([FromBody]SaveETRAMeetingInputModel inputM
         /// Gets an ETRA meeting
         /// </summary>
         /// <returns>A JSON object for a successfully created ETRA meeting request</returns>
-        [HttpGet]
+        [HttpPatch]
```
Comment (bjpirt) [inline ManageATenancyAPI/Controllers/v2/TraController.cs]: Not sure about this - won't it stop the GET request from working?

File: ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs [hunk 2]
```diff
@@ -444,8 +444,8 @@ public async Task<object> GetTenancyManagementInteraction(string contactId, stri
                 _client = _hackneyAccountApiBuilder.CreateRequest(token).Result;
                 _client.DefaultRequestHeaders.Add("Prefer", "odata.include-annotations=\"OData.Community.Display.V1.FormattedValue\"");
 
-                var query = HousingAPIQueryBuilder.getTenancyInteractionDeatils(contactId, personType);
-
+                  var query = HousingAPIQueryBuilder.getTenancyInteractionDeatils(contactId, personType);
+               // var query = HousingAPIQueryBuilder.updateInteractionQuery11();
```
Comment (bjpirt) [inline ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs]: May as well remove this I think

File: ManageATenancyAPI/Controllers/v2/TraController.cs (hunk 1 above)
Comment (lbhssandilya) [reply ManageATenancyAPI/Controllers/v2/TraController.cs]: The method makes  a Patch request and  has a HttpGet  header which was stopping swagger to load ,the method is public async Task<IActionResult> Patch([FromBody]SignOffMeetingInputModel inputModel)

File: ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs (hunk 2 above)
Comment (lbhssandilya) [reply ManageATenancyAPI/Actions/Housing/NHO/TenancyManagementActions.cs]: Removed it, Thanks

**Pull request merged**